internal/dot/graph.go

```diff
@@ -20,34 +20,27 @@
 
 package dot
 
+// Ctor encodes the edges that connects results to params in a graph. It includes information
+// like the name of the constructor, the package, file, line where the constructor is
+// provided, and the params and results of the constructor.
+type Ctor struct {
+	Name    string
+	Package string
+	File    string
+	Line    int
+	Params  []*Node
+	Results []*Node
+}
+
+// Graph is the DOT-format graph in a Container represented by a list of Ctors.
```

Comment:
Same as before: Remove the "list of Ctors" part since that's just listing the field name.